cachi2/core/package_managers/pip.py

```diff
@@ -89,11 +89,11 @@ class DistributionPackageInfo:
 
     name: str
     version: str
-    package_type: Literal["sdist", "wheel"]
     path: Path
     url: str
     is_yanked: bool
 
+    package_type: Literal["sdist", "wheel"] = "sdist"
```

Comment:
I am not sure if this is necessary since we are setting it anyway in
`_process_package_distributions`

Reply:
Agreed. Probably a relic of trying to figure out how to work around VCS/URL reqs not having this property.

Reply:
Btw. why is the `package_type` attribute moved at the end of the class ?

Reply:
I know I had a reason, but honestly, I can't remember at this point.

Reply:
still no idea about this one ?

Reply:
I'm curious about this as well. The change of order in the inputs to the DPI class resulted in quite a few other changes to the tests, etc. If it isn't necessary for some functional reason, I think we should keep it as it was to reduce the noise